packages/lib/rateLimit.ts

```diff
@@ -57,6 +57,12 @@ export function rateLimiter() {
       prefix: "ratelimit",
       limiter: Ratelimit.fixedWindow(10, "60s"),
     }),
+    common: new Ratelimit({
```

Comment:
```suggestion
    common: () => new Ratelimit({
```
 We should invoque the limiters otherwise we're loading all of them instead of only the ones needed. Not a big deal since there's only 3 but a good practice for scaling

Reply:
Getting some errors doing this - will create issue n follow up!